app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with model: @post, url: new_user_post_path(current_user), method: :post do |form| %>
+  <!-- Form fields -->
+  <div class="form-group">
+    <%= form.label :title %>
+    <%= form.text_field :title, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= form.label :text %>
+    <%= form.text_area :text, class: "form-control" %>
+  </div>
+  <%= form.submit 'Create Post', class: "btn btn-primary" %>
+<% end %>
```

Comment:
- [ ] On submit of form I had an error, and this also hindered me from testing comments and likes, kindly fix this
![Screenshot from 2023-11-03 18-30-54](https://github.com/diarisdiakite/Blog-app/assets/66097371/b1fb1b85-d835-4366-aa1a-b634e0de87c2)

Reply:
Thank you @VICTOR-CHUKWU for the comment. The code has been updated.